Library/Formula/websockify.rb

```diff
@@ -0,0 +1,70 @@
+require "FileUtils"
+
+class Websockify < Formula
+  homepage "https://github.com/kanaka/websockify"
+  url "https://github.com/kanaka/websockify/archive/v0.6.0.tar.gz"
+  sha1 "f2bffebc9e4d94ad3a329638bee63e899eaad9c0"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
```

Comment:
Is it part of the other OS X versions as well? If it isn't you may need to create a conditional resource, along the lines of:

```
resource "numpy" do
  url "blah"
  sha1 "blah"
end
```

and then:

```
def install
   if MacOS.version =< :lion
      resource("numpy").stage do
        python.blah
      end
   end
```

Or similar. Tim will probably be able to tell you how widely available ` numpy ` is on the OS X versions.